WPTRT/theme-sniffer
# Development

This is the production release PR.

We would like to release the plugin to the wp.org repo so we need to do a final check if everything works, if the readme is correct, etc.

Once this is done, a master branch will be tagged and released and I can submit the plugin in the repo.

## How to test

Clone the repo in your plugins folder and switch to the `development` branch

```bash
git clone git@github.com:WPTRT/theme-sniffer.git && git checkout development
```

After that run

```bash
cd theme-sniffer && composer install && npm install && npm run build
```

This should make the plugin operational (plus create a zip file for production but that's not important as it won't be bundled in the final release).


@ernilambar @grappler If you can review the code, and if all is ok approve it that would be great 🙂

@timelsass Since you've made some great contributions it would be great if you could do a sanity check.

@pattonwebz @kafleg @joyously @jocastaneda @justintadlock  You can also give it a go and report issues if found.


⚠️Notice: Can anybody who is admin check what I messed with travis, since I no longer trigger the travis checks 😕
